backend/app/routes/admin/updateAdmin.js

```diff
@@ -0,0 +1,36 @@
+const to = require('await-to-js').default;
+const Admin = require('../../models/Admin');
+const { ErrorHandler } = require('../../../helpers/error');
+const constants = require('../../../constants');
+
+module.exports = async (req, res, next) => {
+  const userRecord = await Admin.findOne({
```

Comment:
Why do you need this? Please use only 1 DB Call

Reply:
Will make the required changes